graphql/util.go

```diff
@@ -230,6 +230,7 @@ func SchedulePatch(ctx context.Context, patchId string, version *model.Version,
 		return errors.Wrap(err, "Error setting description"), http.StatusInternalServerError, "", ""
 	}
 
+	ctx := context.Background()
```

Comment:
I think we should comment why this context is defined in the function, so it's not seen as an oversight and switched back. (Could be added to the function comment)

Reply:
added